### ISSUE TITLE
Telegram: ответы в чатах показываются некорректно

### ISSUE THREAD

**Issue by dartraiden** [miranda-ng/miranda-ng#4301]
Когда в чате Вася делает ответ на реплику Пети, то в ответе показано, что это ответ не на реплику Пети, а на реплику от имени чата

<img width="474" alt="mstsc_nbdjA4K1vr" src="https://github.com/miranda-ng/miranda-ng/assets/5046855/85fc5294-a58c-4250-b784-d5887a34cae9">

**Comment by georgehazan** [comment]
может он и пишет от имени чата?

**Comment by dartraiden** [comment]
Нет, воспроизводится даже в нашем тестовом чате. Просто процитируй кого-нибудь через оф. клиент, а затем увидишь, что в Миранде это отобразилось некорректно.
